extension/src/options.js

```diff
@@ -141,10 +141,43 @@ function defaultOptions(): StoredOptions {
         // TODO add docs on positioning
         // TODO eh, would be nice to make it work with --right: true. right now it doesn't
         position_css: `
+		@import url('https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;600&family=PT+Serif:ital,wght@0,400;0,700;1,400;1,700&display=swap');
```

Comment:
No, sidebar uses it )
Importing font implemented in iframe scope (as it should), but some sites (for. ex. github) have warnings, fonts not loaded:
`Refused to load the stylesheet 'https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;600&family=PT+Serif:ital,wght@0,400;0,700;1,400;1,700&display=swap' because it violates the following Content Security Policy directive: "style-src 'unsafe-inline' github.githubassets.com". Note that 'style-src-elem' was not explicitly set, so 'style-src' is used as a fallback.`

So font duplicated in options, b'cose their css scope = initial page scope, no policy warnings.